closure-typing/calculator.js

```diff
@@ -1,5 +1,44 @@
-function calculator() {
-  // your code here
-}
+function calculator(userInput) {
+  var operator = '';
+  var posOfOperator;
+  var result;
+  var firstNumber, secondNumber;
+
+  // Search of operator's type and position
+  for(let i = 0; i < userInput.length; i++) {
+    if (userInput[i] == '+' || userInput[i] == '-' || userInput[i] == '*' || userInput[i] == '/') {
+      operator = userInput[i];
+      posOfOperator = i;
+    }
+  }
+  // Divide userInput upto two operators
+  firstNumber = +userInput.substring(0, posOfOperator);
+  secondNumber = +userInput.substring(posOfOperator + 1, userInput.length);
+  
 
+  // Main logic
+  // some new comments
+  function ZeroDivisionException(firstNumber, secondNumber){
```

Comment:
Nice that you thought about exception, also smth like checkForException is better naming, just to understand what it does